src/LinuxTracing/SwitchesStatesNamesVisitor.cpp

```diff
@@ -112,11 +113,12 @@ void SwitchesStatesNamesVisitor::Visit(uint64_t event_timestamp,
                            event_data.was_created_by_pid);
 }
 
-void SwitchesStatesNamesVisitor::Visit(uint64_t event_timestamp,
-                                       const SchedSwitchPerfEventData& event_data) {
-  // Note that context switches with tid 0 are associated with idle CPU, so we never consider them.
-
+template <typename SchedSwitchPerfEventDataT>
+void SwitchesStatesNamesVisitor::VisitSchedSwitch(uint64_t event_timestamp,
+                                                  const SchedSwitchPerfEventDataT& event_data,
+                                                  bool has_switch_out_callstack) {
   // Process the context switch out for scheduling slices.
+  // Note that context switches with tid 0 are associated with idle CPU, so we never consider them.
```

Comment:
I think that this sentence is imprecise, because a context switch involves two threads, and one of the two being idle CPU doesn't mean discarding the whole event. Maybe something like "Note that tid 0 is associated with idle CPU, so we never consider it."

Reply:
Sounds good. Done.